### PR TITLE
Various bug fixes, config option to disable all ore variants in void miner

### PR DESCRIPTION
**Pull request by hjae78**

### CODE REVIEW

File: src/main/java/gregicadditions/recipes/RecipeHandler.java [hunk 1]
```diff
@@ -285,42 +285,52 @@ private static void processWireGt(OrePrefix wireGt, IngotMaterial material) {
                 case 0:
                 case 1:
                     RecipeMaps.ASSEMBLER_RECIPES.recipeBuilder().circuitMeta(24 + cableSize).input(OrePrefix.wireGtSingle, material, cableAmount).input(foil, Rubber, cableAmount).outputs(cableStack).duration(150).EUt(8).buildAndRegister();
+                    break;
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/RecipeHandler.java:288]: Here the non break is expected. I would like that lower tier recipe can be made with higher tier recipe

File: src/main/java/gregicadditions/recipes/RecipeHandler.java [hunk 2]
```diff
@@ -285,42 +285,52 @@ private static void processWireGt(OrePrefix wireGt, IngotMaterial material) {
...
                     RecipeMaps.ASSEMBLER_RECIPES.recipeBuilder().circuitMeta(24 + cableSize).input(OrePrefix.wireGtSingle, material, cableAmount).input(foil, PolyphenyleneSulfide, cableAmount).outputs(cableStack).duration(150).EUt(8).buildAndRegister();
-
+                    break;
             }
         }
         switch (tier) {
             case 0:
             case 1:
                 RecipeMaps.ASSEMBLER_RECIPES.recipeBuilder().circuitMeta(24).input(wireGt, material).input(foil, Rubber, cableAmount).outputs(cableStack).duration(150).EUt(8).buildAndRegister();
+                break;
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/RecipeHandler.java:314]: Here is the same comment. Non break it's expected

**Comment by hjae78** [comment]
Ah that make sense, will push

**Comment by huneau** [comment]
I finally merge the PR I let you close all related issue.

**Pull request merged**